### PR TITLE
Editor beta: first draft

### PR DESCRIPTION
**Pull request by davidkpiano**

### CODE REVIEW

**Comment by farskid** [comment]
Reads well so far 👍

File: content/posts/2022-2-6-stately-editor-public-beta.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+author:
+  - David Khourshid
+originalURL: ""
+excerpt: ""
+publishedAt: 2022-2-6
+---
+
+We're excited to announce the public beta of the Stately Editor! The Stately Editor is a tool for creating and editing state diagrams. We've received a lot of great feedback from the private beta testers, and now we look forward to sharing it with everyone.
+
+The software development lifecycle today is a bit fragmented. It's very difficult for user stories, designs, code, tests, and documentation to stay in sync, leading to things going out-of-date, or even missing completely. Bugs increase, and adding or modifying features becomes increasingly difficult. And even understanding all the features and use-cases of an app becomes a daunting task.
```
Comment (laurakalbag) [inline content/posts/2022-2-6-stately-editor-public-beta.md:16]:
```suggestion
The software development lifecycle today is a bit fragmented. It’s very difficult for user stories, designs, code, tests, and documentation to stay in sync, leading to parts going out-of-date or even missing entirely. Bugs increase, and adding or modifying features becomes increasingly complex. And even understanding all the features and use-cases of an app becomes a daunting task.
```

File: content/posts/2022-2-6-stately-editor-public-beta.md [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+
+Our mission is to make app logic accessible with the entire team, including developers, designers, and stakeholders. By using state machines and statecharts as a common visual language, the software development cycle becomes more collaborative, which eliminates handoff and tightens the feedback loop.
```
Comment (laurakalbag) [inline content/posts/2022-2-6-stately-editor-public-beta.md:18]:
```suggestion
Our mission is to make app logic accessible to the entire team, including developers, designers, and stakeholders. Using state machines and statecharts as a common visual language, the software development cycle becomes more collaborative, eliminating handoff and tightening the feedback loop.
```

File: content/posts/2022-2-6-stately-editor-public-beta.md [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+
+The Stately Editor aims to be that collaborative visual app logic tool. Before, developers could create state machines and statecharts in code using [XState](https://github.com/statelyai/xstate). Now, those same state machines and statecharts can be created visually.
```
Comment (laurakalbag) [inline content/posts/2022-2-6-stately-editor-public-beta.md:20]:
```suggestion
The Stately Editor aims to be that collaborative visual app logic tool. Before, developers could build state machines and statecharts in code using [XState](https://github.com/statelyai/xstate). Now, you can create those same state machines and statecharts visually.
```

File: content/posts/2022-2-6-stately-editor-public-beta.md [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+
+And yes, it goes both ways: edit the code, and see the updated statechart. Edit the statechart diagram, and see the updated code. All in real-time.
```
Comment (laurakalbag) [inline content/posts/2022-2-6-stately-editor-public-beta.md:22]:
```suggestion
And yes, it goes both ways: edit the code and see the updated statechart. Edit the statechart diagram and see the updated code. All in real-time.
```

File: content/posts/2022-2-6-stately-editor-public-beta.md [hunk 5]
```diff
@@ -0,0 +1,77 @@
...
+
+## Editor overview
+
+To get started quickly, go to [stately.ai/editor](https://stately.ai/editor). From there, you can start creating your machines in the editor. See this video for more information:
```
Comment (laurakalbag) [inline content/posts/2022-2-6-stately-editor-public-beta.md:26]:
```suggestion
To get started quickly, go to [stately.ai/editor](https://stately.ai/editor). From there, you can begin creating your machines in the editor. Watch the video below to find out more:
```

File: content/posts/2022-2-6-stately-editor-public-beta.md [hunk 6]
```diff
@@ -0,0 +1,77 @@
...
+
+> TODO: add video
+
+You can also create new machines in any system, which will open them in the editor:
+
+> TODO: add video
+
+## Features and roadmap
+
+Most statechart features are supported by the editor:
```
Comment (laurakalbag) [inline content/posts/2022-2-6-stately-editor-public-beta.md:36]:
```suggestion
The editor supports most statechart features:
```

File: content/posts/2022-2-6-stately-editor-public-beta.md [hunk 7]
```diff
@@ -0,0 +1,77 @@
...
+
+- Atomic, compound, parallel, history, and final states
+- Normal, guarded, and delayed events/transitions
+- Entry, exit, and transition actions
+- Invocations
+- Done and error transitions for states and invocations
+- State and event descriptions
+
+There's currently two modes:
```
Comment (laurakalbag) [inline content/posts/2022-2-6-stately-editor-public-beta.md:45]:
```suggestion
There are currently two modes:
```

File: content/posts/2022-2-6-stately-editor-public-beta.md [hunk 8]
```diff
@@ -0,0 +1,77 @@
...
+- Markdown generation
+- Event recording and playback
+- Machine versioning and diffing
+- Autolayout and smart layout controls
+
+## Join the public beta
+
+Excited to see what the Stately Editor can do? Join the public beta by registering for free at [stately.ai/registry](https://stately.ai/registry) and creating your first machine at [stately.ai/editor], or [discover other people's machines](https://stately.ai/registry/discover) and get inspired.
+
+If you have any feedback, such as feature requests or bug reports, please [let us know](https://github.com/statelyai/editor-feedback/issues/new).
```
Comment (laurakalbag) [inline content/posts/2022-2-6-stately-editor-public-beta.md:69]:
```suggestion
Please [let us know](https://github.com/statelyai/editor-feedback/issues/new) if you have any feedback, feature requests or bug reports.
```

File: content/posts/2022-2-6-stately-editor-public-beta.md [hunk 9]
```diff
@@ -0,0 +1,77 @@
...
+
+> TODO: CTA?
+
+## Next steps
+
+This is our first major step into making the ultimate collaborative tool for visually editing even the most complex app logic. We're excited to share the Stately Editor with the world, and we're looking forward to hearing your feedback.
```
Comment (laurakalbag) [inline content/posts/2022-2-6-stately-editor-public-beta.md:75]:
```suggestion
The editor is our first significant step towards making the ultimate collaborative tool for visually editing even the most complex app logic. We’re excited to share the Stately Editor with the world, and we’re looking forward to hearing your reactions.
```

File: content/posts/2022-2-6-stately-editor-public-beta.md [hunk 10]
```diff
@@ -0,0 +1,77 @@
...
+
+Make sure to [join our Discord community](https://discord.gg/xstate) to get help, ask questions, and get feedback.
```
Comment (laurakalbag) [inline content/posts/2022-2-6-stately-editor-public-beta.md:77]:
```suggestion
Make sure you [join our Discord community](https://discord.gg/xstate) to get help, ask questions, and give us your feedback.
```

**Review by laurakalbag — COMMENTED** [review]
Love it! Added some minor text edits for flow.

File: content/posts/2022-2-6-stately-editor-public-beta.mdx [hunk 11]
```diff
@@ -0,0 +1,75 @@
+---
+title: Stately Editor Public Beta
+description: asdf
+tags:
+  - editor
+  - announcement
+author: David Khourshid
+originalURL: ""
+excerpt: ""
+publishedAt: 2022-2-6
```
Comment (farskid) [inline content/posts/2022-2-6-stately-editor-public-beta.mdx:10]: @davidkpiano Change this date before merging tomorrow.

File: content/posts/2022-2-6-stately-editor-public-beta.mdx [hunk 12]
```diff
@@ -0,0 +1,78 @@
...
+
+Our mission is to **make app logic accessible to the entire team**, including developers, designers, and stakeholders. Using state machines and statecharts as a common visual language, the software development cycle becomes more collaborative, eliminating handoff and tightening the feedback loop.
+
+The Stately Editor aims to be that collaborative visual app logic tool. Before, developers could build state machines and statecharts in code using [XState](https://github.com/statelyai/xstate). Now, you can create those same state diagrams visually.
+
+And yes, it goes both ways: edit the code and see the updated state diagram. Edit the state diagram and see the updated code in real-time. All this is possible with the [XState VS Code extension](https://marketplace.visualstudio.com/items?itemName=mattpocock.xstate-vscode&ssr=false).
```
Comment (farskid) [inline content/posts/2022-2-6-stately-editor-public-beta.mdx:25]: Extension link is broken.

File: content/posts/2022-2-6-stately-editor-public-beta.mdx [hunk 13]
```diff
@@ -0,0 +1,78 @@
...
+- Teams and private machines/systems
+- _And so much more._
+
+## Join the public beta
+
+Excited to see what the Stately Editor can do? Join the public beta by registering for free at [stately.ai/registry](https://stately.ai/registry?source=blog) and creating your first machine at [stately.ai/editor](stately.ai/editor?source=blog), or [discover other people’s machines](https://stately.ai/registry/discover?source=blog) and get inspired.
+
+Please [let us know](https://github.com/statelyai/editor-feedback/issues/new) if you have any feedback, feature requests or bug reports.
+
+<Announcement cta="Sign up here">
```
Comment (farskid) [inline content/posts/2022-2-6-stately-editor-public-beta.mdx:70]: We need an `href` here.

Comment (laurakalbag) [reply content/posts/2022-2-6-stately-editor-public-beta.mdx]: https://stately.ai/registry ? That’ll take you to sign in. Also text could be clearer if it said “Sign up for the beta”.

File: content/posts/2022-2-6-stately-editor-public-beta.mdx [hunk 14]
```diff
@@ -0,0 +1,78 @@
...
+  - announcement
+  - beta
+author: David Khourshid
+ogImage: "https://stately.ai/blog/2022-02-08-join-the-stately-editor-public-beta.png"
+publishedAt: 2022-2-8
+---
+
+We’re excited to announce the public beta of the Stately Editor! The Stately Editor is a tool for creating and editing state diagrams. We’ve received a lot of great feedback from the private beta testers, and now we’re delighted to share it with everyone.
+
+**Try the Stately Editor beta now at [stately.ai/editor](https://stately.ai/editor)**
```
Comment (laurakalbag) [inline content/posts/2022-2-6-stately-editor-public-beta.mdx:15]:
```suggestion
**[Try the Stately Editor beta now](https://stately.ai/editor)**
```

File: content/posts/2022-2-6-stately-editor-public-beta.mdx [hunk 15]
```diff
@@ -0,0 +1,78 @@
...
+- Markdown generation
+- Event recording and playback
+- Machine versioning and diffing
+- Autolayout and smart layout controls
+- Teams and private machines/systems
+- _And so much more._
+
+## Join the public beta
+
+Excited to see what the Stately Editor can do? Join the public beta by registering for free at [stately.ai/registry](https://stately.ai/registry?source=blog) and creating your first machine at [stately.ai/editor](https://stately.ai/editor?source=blog), or [discover other people’s machines](https://stately.ai/registry/discover?source=blog) and get inspired.
```
Comment (laurakalbag) [inline content/posts/2022-2-6-stately-editor-public-beta.mdx:66]:
```suggestion
Excited to see what the Stately Editor can do? [Join the public beta by registering for free](https://stately.ai/registry?source=blog) and [create your first machine](https://stately.ai/editor?source=blog), or [discover other people’s machines](https://stately.ai/registry/discover?source=blog) and get inspired.
```

File: content/posts/2022-2-6-stately-editor-public-beta.mdx [hunk 16]
```diff
@@ -0,0 +1,78 @@
...
+
+Please [let us know](https://github.com/statelyai/editor-feedback/issues/new) if you have any feedback, feature requests or bug reports.
+
+<Announcement cta="Sign up here" href="https://stately.ai/editor?source=blog">
```
Comment (laurakalbag) [inline content/posts/2022-2-6-stately-editor-public-beta.mdx:70]:
```suggestion
<Announcement cta="Sign up for the beta" href="https://stately.ai/editor?source=blog">
```

**Pull request merged**